### PR TITLE
Steering committee notes for 2017

### PR DESCRIPTION
**Pull request by niloc132**
Suggesting we make this the canonical location to store notes, rather than individual (hard to search) docs linked to a mailing list. All added notes have anchors now for easy external linking to specific meetings, like release notes have.

---
This change is [Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/248)

### CODE REVIEW

File: src/main/markdown/meetingnotes.md [hunk 1]
```diff
@@ -1,7 +1,63 @@
 Steering Committee Meeting Notes
 ================================
 
-## 07/20/2016
+## <a id="05/10/2017"></a> 05/10/2017
```
Comment (tbroyer) [inline src/main/markdown/meetingnotes.md]: Please use ISO 8601 (at least for the IDs); MDY is foreign/weird in most of the world.

https://en.wikipedia.org/wiki/Date_format_by_country

**Comment by niloc132** [comment]
Sure - in both page splitting and date formatting, I went with the (approved by you, I just re-read and chatted with the author) earlier PR and tried to remain consistent.

Will change this to make up a TOC page, split pages, and edit dates to yyyy-mm-dd. 

For the existing content before I started owning note-taking, the content came from the gwt-steering mailing list, anything else we'll need to get from Bhaskar directly. Let's ask on wednesday.

**Comment by niloc132** [comment]
Files are split, date formats should be easier to manage now.

File: src/main/markdown/meetingnotes.md [hunk 2]
```diff
@@ -1,137 +1,16 @@
...
-    * Plan is to super-cede GPE with GWT plug-in
-    * Both plug-ins should work side-by-side
-    * Why is GWT GPE plug-in different from other plug-ins? Should it be externally hosted similar to maven and gradle?
-    * Decision on this deferred.
+* <a href="meetingnotes/2017-05-10.html">10 May 2017</a>
+* <a href="meetingnotes/2017-04-12.html">12 April 2017</a>
+* <a href="meetingnotes/2017-03-08.html">08 March 2017</a>
+* <a href="meetingnotes/2017-02-08.html">08 February 2017</a>
+* <a href="meetingnotes/2017-01-11.html">11 January 2017</a>
+* <a href="meetingnotes/2016-07-20.html">20 July 2016</a>
```
Comment (tbroyer) [inline src/main/markdown/meetingnotes.md]: Should we add a "to be published" or "…" line here to at least hint that there have been meetings in this period?

**Comment by tbroyer** [comment]
Let's merge as-is and improve later.

**Pull request merged**